etrago/tools/utilities.py

```diff
@@ -287,9 +290,24 @@ def buses_by_country(self, apply_on="grid_model"):
     con = self.engine
     germany_sh = gpd.read_postgis(query, con, geom_col="geometry")
 
-    path = gpd.datasets.get_path("naturalearth_lowres")
-    shapes = gpd.read_file(path)
-    shapes = shapes[shapes.name.isin([*countries])].set_index(keys="name")
+    # read Europe borders. Original data downloaded from naturalearthdata.com/
+    # under Public Domain license
+    path_countries = Path(".") / "tools" / "shapes_europe"
```

Comment:
There is already a sub-directory called "data" in etrago. I would either store the file in that folder or in the working directory.

Reply:
Sure. The path was changed in 712eda0ba489fc31c33dcd7f5cf4449abffa1609.